src/edit.js

```diff
@@ -2,12 +2,16 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Placeholder } from '@wordpress/components';
+import { Disabled } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Calendar from './frontend/app';
 
 const EditView = () => (
-	<Placeholder
-		icon="calendar"
-		label={ __( 'Go to "Meetings" to add or edit your meetings', 'wporg' ) }
-	/>
+	<Disabled>
```

Comment:
disabling interactive blocks within the editor is kinda a force of habit now 😅since some interaction depends on different contexts, I don’t think this is the case in this block so I might remove the disabled wrap, however, it might be confusing to the user if they think the changes they make in the editor will persist on the frontend (changing default month or changing display style).